### DISCUSSION TITLE
The Insert, Update, Delete commands

### DISCUSSION THREAD

**Discussion by YaroslavTsvirkun** [demmings/gsSQL#60] [Q&A]
Hi, 
And the Insert, Update, Delete commands will work interestingly?

**Comment by demmings** [comment]
The gsSQL() function is strictly an SQL SELECT.  A custom function does not have rights to update cells in the spreadsheet.  It can only return data as a single value or an array of arrays.  I just wanted something easier to use than the QUERY() command.
